framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -23,11 +23,10 @@ Licensed to the Apache Software Foundation (ASF) under one
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
-import org.apache.cordova.CordovaInterface;
-import org.apache.cordova.CordovaPlugin;
-import org.apache.cordova.LOG;
 import org.json.JSONException;
 import org.json.JSONObject;
+import org.xwalk.core.XWalkActivity;
+import org.xwalk.core.XWalkCoreWrapper;
```

Comment:
It seems that XWalkCoreWrapper is not used. I also do not recommend to use it outside of XWalkAcvitiy.
XWalkCoreWrapper is not designed to be part of Embedding API, so will not be exposed to external in the future.